Bug with Choosing File and Folder Dialogs

SirLouen opened the pull request:
At some point during #24 I deleted these two lines. 

Ideally merge this from #26 but can be merged from #24 without conflicts.

schorschii commented:
> At some point during #24 I deleted these two lines.

Really? Which lines...? I lost the overview of your PRs. If you accidentally deleted lines in #24, you can add another commit to the branch to fix it and it will appear in the same PR.

SirLouen commented:
> > At some point during #24 I deleted these two lines.
> 
> Really? Which lines...? I lost the overview of your PRs. If you accidentally deleted lines in #24, you can add another commit to the branch to fix it and it will appear in the same PR.

I mean, while working on PR #24 I deleted those lines, and when I sent the final push it didn't contain these so I created this PR because #23 existed.

I have sorted the merging conflict so theorically you could add #24  and then this.

SirLouen commented:
I've sorted this in #24
